index.html

```diff
@@ -1169,19 +1169,77 @@ <h2>Relative DID URLs</h2>
 
   </section>
 
-  <section class="informative">
+  <section>
     <h1>Data Model</h1>
-
+    <p>
+This specification defines an abstract data model for <a>DID documents</a>,
+independent of any specific representation. This section provides a
+high-level description of the data model, a set of requirements for
+representations, and a set of requirements for extensibility.
+    </p>
     <section>
       <h2>Definition</h2>
+      <p>
+A <a>DID document</a> consists of a collection of name-value pairs which will be
+referred to as properties in this document. The definitions of each of these
+properties are specified in section <a href="#core-properties"></a>. Specific
+representations are defined in section <a href="#core-representations"></a>.
+      </p>
     </section>
-
     <section>
-      <h2>Extensibility</h2>
+      <h2>Representations</h2>
+      <p>
+Following are the requirements for conformant representations.
+      </p>
+      <ol>
+        <li>
+A representation MUST define an unambiguous encoding and decoding of all
+properties defined in this specification. This definition MUST enable lossless
+conversion of the value of any property into another representation. This means
+anything you can represent in the DID document model can be represented in any
+compliant representation.
+        </li>
+        <li>
+The representation MUST have an unambiguous MIME type.
```

Comment:
```suggestion
The representation MUST be associated with a MIME type.
```

Not sure what an "unambiguous" MIME type is... does that mean something like "text/plain" vs "text/html"... where text/plain is more ambiguous than text/html?

Reply:
Should we say the MIME type should be IANA registered or is that overly constraining?

Reply:
> Should we say the MIME type should be IANA registered or is that overly constraining?

Imho, yes, we should.